examples/llm_compression/openvino/tiny_llama_find_hyperparams/README.md

```diff
@@ -0,0 +1,36 @@
+# Find the appropriate hyperparameters to compress the TinyLLama model
+
+This example demonstrates how to find the appropriate `ratio` and `group_size` parameters to compress the weights of the TinyLLama model from the HuggingFace Transformers. OpenVINO backend supports inference of mixed-precision models with weights compressed to a 4-bit data type as a primary precision. The fastest mixed-precision mode is `INT4_SYM`, but it may lead to a significant accuracy degradation, especially for models of moderate size. In this example, the allowed maximum deviation from the original model is `0.2` points of the similarity metric. If the similarity of the compressed model is not satisfying, there are 2 hyper-parameters to tune: `group_size` and `ratio`. Smaller `group_size` and `ratio` of 4-bit layers usually improve accuracy at the sacrifice of model size and inference latency.
+
+The example includes the following steps:
+
+- Prepare `wikitext` dataset.
+- Prepare `TinyLlama/TinyLlama-1.1B-step-50K-105b` text-generation model in OpenVINO representation using Optimum-Intel.
+- Compress weights of the model with NNCF Weight compression algorithm.
+- Find appropriate `ratio` and `group_size` if acceptable similarity is not achieved.
+- Measure the similarity and footprint of the final model.
+
+## Install requirements
+
+To run the example you should install the corresponding Python dependencies:
+
+- Create a separate Python* environment and activate it: `python3 -m venv nncf_env && source nncf_env/bin/activate`
+- Install dependencies:
+
+```bash
+pip install -U pip
+pip install ../../../../
+pip install -r requirements.txt
+```
+
+## Run Example
+
+The example is fully automated. Just run the following command in the prepared Python environment:
+
+```bash
+python main.py
+```
+
```

Comment:
Since this is an example it would be helpful to explain how to plug a different model model. Maybe it would be better to place comments about this in the code as well.